react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work correctly!
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+  const [backgroundColor, setBackgroundColor] = useState('white');
+
+  return (
+    <div>
+      <div>{items[current]}</div>
+      <div>
+        <Button text="Prev"
+          onClick={() => setCurrent(((current - 1) + items.length) % items.length)}
+          style={() => setBackgroundColor(backgroundColor)}
```

Comment:
This doesn't do anything since the `Button` component doesn't have a prop named 'style'. Also, you wouldn't normally pass an arrow function to the `style` prop. It expects an object with keys that are CSS style properties.

Once you remove this, you'll find that the `backgroundColor` state isn't used anymore and can be deleted. It's not necessary to get this exercise working.

Reply:
Removed the `style` and the `useSet` for `backgroundColor`

Reply:
<img width="944" alt="Screenshot 2023-03-31 at 2 09 01 PM" src="https://user-images.githubusercontent.com/110208589/229231786-0b9b5fb9-daf6-4585-8977-962ca35df3a8.png">